src/parquet/types.h

```diff
@@ -283,6 +296,11 @@ PARQUET_EXPORT std::string FormatStatValue(Type::type parquet_type, const char*
 
 PARQUET_EXPORT int GetTypeByteSize(Type::type t);
 
+SortOrder::type PARQUET_EXPORT DefaultSortOrder(Type::type primitive);
+
+SortOrder::type PARQUET_EXPORT GetSortOrder(LogicalType::type converted,
```

Comment:
Can you put the `PARQUET_EXPORT` before the return type for consistency (we should go through and make them all consistent)?